src/win/tty.c

```diff
@@ -1176,13 +1185,35 @@ static int uv_tty_set_style(uv_tty_t* handle, DWORD* error) {
   char fg_bright = -1, bg_bright = -1;
   char inverse = -1;
 
+  char default_fg_color = 7, default_bg_color = 0;
+  char default_fg_bright = 0, default_bg_bright = 0;
+  char default_inverse = 0;
+
+  if (uv_tty_default_text_attributes != 0) {
```

Comment:
If y'all don't mind a new function and static data, I'm certainly OK doing it that way.

I was trying to keep the changes localized so the patch would be easier to review.  I performed minimally invasive surgery on a code base where I'm not familiar with all the local customs and traditions. :-)